clusterloader2/pkg/measurement/common/probes/manifests/metricsServer/metrics-server-prober-deployment.yaml

```diff
@@ -4,27 +4,27 @@ metadata:
   namespace: probes
   name: metrics-server-prober
   labels:
-    probe: metrics-server-probe
+    probe: kube-client
```

Comment:
Hmm - I actually likes the metrics-server-probe (or sth like that).

kube-client seems too generic too me.

Why are you changing this?

Reply:
yeah, thanks for review. I have updated all suggestions